lib/ziti-tunnel-cbs/include/ziti/ziti_dns.h

```diff
@@ -20,6 +20,7 @@
 #include <ziti/ziti_tunnel.h>
 
 #define DNS_NO_ERROR 0
+#define DNS_SERVFAIL 2
```

Comment:
Will this change how ZET's nameserver responds to hits or misses?

Reply:
Actually this change is benign as no code uses the SERVFAIL symbol. I didn't intend to include this file in the comment, and it will be backed out shortly.